proposals/remove-prefix-from-config.md

```diff
@@ -0,0 +1,180 @@
+<!--
+Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"). You may
+not use this file except in compliance with the License. A copy of the
+License is located at
+
+http://aws.amazon.com/apache2.0/
+
+or in the "license" file accompanying this file. This file is distributed
+on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied. See the License for the specific language governing
+permissions and limitations under the License.
+-->
+
+# Introduction
+As the CLI moves to multiple cluster configurations, the prefixes in the old configuration will be removed because:
+1. Bad user experience and not really a cluster configuration.
+1. Every new customers need to understand what the prefixes are for in order to know the names of the resources created by ECS CLI
+1. Customers do not have the flexibility to use the resource name they wish to have unless they set the prefixes to empty string.
+
+The 3 prefixes that are used in the configuration are:
+* **compose-project-name-prefix** - name used for create task definition (<compose_project_name_prefix> + <project_name>)
+* **compose-service-name-prefix** - name used for create service (<compose_service_name_prefix> + <project_name>)
+* **cfn-stack-name-prefix** - name used for creating CFN stacks (<cfn_stack_name_prefix> + <cluster_name>)
+
+# Solution
+When users run `ecs-cli configure`, the prefixes will be removed from the config.
+```yaml
+default: my_config
+clusters:
+  my_config:
+  - cluster: demo
+  - region: us-west-2
+```
+
```

Comment:
I think it's a good idea to simplify this to use cluster name as the configuration name, however, this would become a problem if someone wish to configure the same cluster in a different region.